client/src/components/Header/Header.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import Header from "./Header";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("Header Component Tests", () => {
+  const props = {};
+
+  it("Header renders correctly", () => {
```

Comment:
Add tests:
1. Render title and subtitle
2. Button performs action on click

Reply:
done in #159